### PR TITLE
Scarr1run

### PR DESCRIPTION
**Pull request by GaryQ-physics**
theres a few big changes in the overall structure

### CODE REVIEW

File: cutplane/probe.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def probe(time, P, var=None, debug=False, dictionary=False, usekV=False):
     if type(time) == str:
         filename = time
     else:
+        print('hello there')
```
Comment (rweigel) [inline cutplane/probe.py]: Hi.

And remove.

File: misc/store_np_arrays_binary.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 import time
 import numpy as np
 
-N = 50000
+N = 50000000
 fname = '/tmp/np_array_test.bin'
```
Comment (rweigel) [inline misc/store_np_arrays_binary.py]: I've been using "/tmp" out of lazyness. There is a python function to get the system tmp directory: https://stackoverflow.com/questions/847850/cross-platform-way-of-getting-temp-directory-in-python

File: util/util.py [hunk 3]
```diff
@@ -90,6 +90,53 @@ def time2filename(time, extension='.out.cdf', split=False):
     return conf["run_path"] + filename
 
 
+def time2CDFfilename(run, time, split=False):
+    """
+    >>> u.time2CDFfilename('SCARR5',[2003,11,20,7,7,0])
```
Comment (rweigel) [inline util/util.py:95]: Include import statement needed for this to run.

**Comment by rweigel** [comment]
I added a few comments.

**Comment by GaryQ-physics** [comment]
ok

**Pull request merged**